SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetBitReader.cpp

```diff
@@ -37,42 +37,24 @@ void FSpatialNetBitReader::DeserializeObjectRef(FUnrealObjectRef& ObjectRef)
 		ObjectRef.Outer = FUnrealObjectRef();
 		DeserializeObjectRef(*ObjectRef.Outer);
 	}
+
+	SerializeBits(&ObjectRef.bNoLoadOnClient, 1);
```

Comment:
This wasn't getting saved previously?

Reply:
Seems so. I can only imagine that we were getting lucky that nothing in replicated structs was pointing at streaming level actors/objects that weren't loaded.